package.json

```diff
@@ -1,17 +1,18 @@
 {
   "name": "applicationinsights",
   "license": "MIT",
-  "bugs": "https://github.com/Microsoft/AppInsights-node.js/issues",
+  "bugs": "https://github.com/Microsoft/ApplicationInsights-node.js/issues",
   "version": "0.12.5",
```

Comment:
If there are breaking changes, shouldn't there be an increment of some variety for this release?